docs/templates/legacy-template.md

```diff
@@ -0,0 +1,13 @@
+# LegacyTemplate
+
+[Source file](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/src/BlockTypes/LegacyTemplate.php)
+
+The LegacyTemplate is a class used to set up the Legacy Template block on the server-side, and render the correct template.
+
+## Overview
+
+It is from this file we enqueue the frontend scripts necessary for the product gallery, add to basket etc. functionality to work on the frontend.
```

Comment:
```suggestion
It is from this file we enqueue the front-end scripts necessary for the product gallery, add to basket etc. functionality to work on the front-end.
```

So far we used the hyphenated form of "front-end", so I am suggesting that. However, this sentence sounds somewhat a bit strange. I'm not sure how I would reword it, maybe:

_It is from this file we enqueue the front-end scripts necessary to enable dynamic functionality, such as the product gallery, add to basket etc._

What do you think?